From a97f868a806ab4f1aaab88c72d4b55bfb18cc3cc Mon Sep 17 00:00:00 2001
From: Alexandre Courbot <acourbot@nvidia.com>
Date: Fri, 9 Dec 2016 17:04:16 +0900
Subject: [PATCH 5/8] tegra: open card1 instead of renderD128

Mesa wants to perform a FLINK ioctl to share GPU buffers. This ioctl is
not valid on render-nodes.

This fixes rendering with Weston, but GLX is still broken.
---
 src/gallium/winsys/tegra/drm/tegra_drm_winsys.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gallium/winsys/tegra/drm/tegra_drm_winsys.c b/src/gallium/winsys/tegra/drm/tegra_drm_winsys.c
index 361370d..e213942 100644
--- a/src/gallium/winsys/tegra/drm/tegra_drm_winsys.c
+++ b/src/gallium/winsys/tegra/drm/tegra_drm_winsys.c
@@ -10,7 +10,7 @@
 static struct pipe_screen *
 tegra_create(struct renderonly *ro)
 {
-   int fd = open("/dev/dri/renderD128", O_RDWR | O_CLOEXEC);
+   int fd = open("/dev/dri/card1", O_RDWR | O_CLOEXEC);
 
    if (fd == -1)
       return NULL;
-- 
2.9.3

